### PR TITLE
Release updates, and fixes build

### PR DESCRIPTION
**Pull request by sfshaza2**
Staged:  https://sz-flutter-2.firebaseapp.com/

### CODE REVIEW

File: src/docs/whats-new-archive.md [hunk 1]
```diff
@@ -11,13 +11,167 @@ join the [flutter-announce][] Google group.
 
 [flutter-announce]: https://groups.google.com/forum/#!forum/flutter-announce
 
+
+## **May 6, 2020, Work-From-Home Edition**
```
Comment (timsneath) [inline src/docs/whats-new-archive.md:15]: Small nit, but the headings are extra bold with the `##` and `**` applied. Is that by design?

Comment (sfshaza2) [reply src/docs/whats-new-archive.md]: No, it's cause I'm lazy.  :). Fixed

**Comment by sfshaza2** [comment]
Thanks for your LGTM, @timsneath!

**Comment by sfshaza2** [comment]
At @timsneath's request, forcibly merging.

**Pull request merged**